MaterialDesignThemes.Wpf/ComboBoxPopup.cs

```diff
@@ -115,6 +121,22 @@ public ComboBoxPopupPlacement PopupPlacement
 
         #endregion
 
+        #region PopupPlacementMode property
+
+        public static readonly DependencyProperty PopupPlacementModeProperty
+            = DependencyProperty.Register(nameof(PopupPlacementMode),
+                typeof(PopupPlacementMode),
+                typeof(ComboBoxPopup),
+                new PropertyMetadata(PopupPlacementMode.Auto));
+
+        public PopupPlacementMode PopupPlacementMode
```

Comment:
Can we leverage the existing PopupPlacement property for this behavior rather than creating a new property/enum?

Reply:
I think it is possible, but it would require to replace internal changes to PopupPlacementProperty with some private variable to store current popup placement value. So the direction is not fixed once it is first time calculated.

Maybe one more improvement could be added - when e.g. UP direction is set but there is not enough space it should probably go down anyway. Then it wouldn't be forced direction but lets say preferred direction. What do you think?

I can prepare new PR for it, or can I somehow edit this one with new commit?

Reply:
Yea lets give that a shot. You can simply push changes to your `ComboBoxPopupDirection` branch and this PR will automatically update.